source/user-manual/capabilities: update CIS-CAT documents to support both v3 and v4

hustliyilin opened the pull request:
Fix https://github.com/wazuh/wazuh/pull/12582

Signed-off-by: YiLin.Li <YiLin.Li@linux.alibaba.com>

<!--
This template reflects sections that must be included in new Pull requests.
Contributions from the community are really appreciated. If this is the case, please add the
"contribution" to properly track the Pull Request.

Please fill the table below. Feel free to extend it at your convenience.
-->
<!--
## Community contributions advice

We love our community contributions. First, we work with the numbered branches. The `master` branch is only updated when a new Wazuh release is done. We recommend making PRs from the actual branch. For instance, if Wazuh 3.11.4 is the latest release, the branch to be used is 3.11.

Anyway, if you contribute from the master branch, we will `cherry-pick` your commits to the numerated branch for you. 

Thanks!
-->

## Description

<!--
Add a clear description of how the problem has been solved. 
If your PR closes an issue, please use the "closes" keyword indicating the issue. 
-->

## Checks
- [x] It compiles without warnings.
- [x] Spelling and grammar. 
- [x] Used impersonal speech. 
- [x] Used uppercase only on nouns. 
- [ ] Updated the `redirect.js` script if necessary (check [this guide](https://github.com/wazuh/wazuh-documentation/blob/master/NEW_RELEASE.md)).

<!--
Leave the following note if you made any changes to the redirect.js script. Remove it otherwise.
-->

## Note to the reviewer

This PR includes changes to the `redirect.js` script that need to be included in all production branches.

vikman90 reviewed (changes requested):
@hustliyilin Thank you so much for this PR. 

I've made some suggestions for minor changes. Please review them and apply them if you agree.

Cheers!

hustliyilin commented:
> @hustliyilin Thank you so much for this PR.
> 
> I've made some suggestions for minor changes. Please review them and apply them if you agree.
> 
> Cheers!

Hi @vikman90 ,

Thank you very much. I already modified the codes according to your review.

vikman90 reviewed (approved):
Thank you @hustliyilin.
We're just awaiting another review by @s-ocando, then we will merge both PRs!

Thank you very much for your contribution!
Best regards.

vikman90 commented on source/user-manual/reference/ossec-conf/wodle-ciscat.rst:
```diff
@@ -27,22 +27,25 @@ Main options
++----------------------+------------------------------------------------------------+
+| `java_path`_         | Any valid path                                             |
++----------------------+------------------------------------------------------------+
+| `ciscat_path`_       | CIS-CAT.sh, CIS-CAT.BAT, Assessor-CLI.sh, Assessor-CLI.bat |
++----------------------+------------------------------------------------------------+
+| `ciscat_binary`_     | Any valid path                                             |
```
Seems like if the allowed values column was switched.

```suggestion
| `ciscat_path`_       | Any valid path                                             |
+----------------------+------------------------------------------------------------+
| `ciscat_binary`_     | CIS-CAT.sh, CIS-CAT.BAT, Assessor-CLI.sh, Assessor-CLI.bat |
```

vikman90 reviewed (changes requested):
Hi @hustliyilin, we have this feedback from @s-ocando. Please take a look on that and fix it if you agree.

Best regards.

hustliyilin commented:
> Hi @hustliyilin, we have this feedback from @s-ocando. Please take a look on that and fix it if you agree.
> 
> Best regards.

Hi @vikman90 ,

Thank you very much. I already modified the codes according to your review and rebased to the newest base branch to fix the conflicts.

vikman90 reviewed (approved):
Looks good to me!

hustliyilin commented:
> Looks good to me!

Thanks  very much @vikman90 ! Cloud you review it again @s-ocando ? Thanks a lot.

s-ocando reviewed (approved):
LGTM. Thank you very much @hustliyilin for your contribution!
